rocAL/rocAL/include/augmentations/geometry_augmentations/node_resize_mirror_normalize.h

```diff
@@ -25,26 +25,29 @@ THE SOFTWARE.
 #include "parameter_factory.h"
 #include "parameter_vx.h"
 
-class ResizeMirrorNormalizeNode : public Node
-{
+class ResizeMirrorNormalizeNode : public Node {
 public:
     ResizeMirrorNormalizeNode(const std::vector<Tensor *> &inputs, const std::vector<Tensor *> &outputs);
     ResizeMirrorNormalizeNode() = delete;
-    void init(std::vector<float>& mean,  std::vector<float>& std_dev, IntParam *mirror);
+    void init(unsigned dest_width, unsigned dest_height, RocalResizeScalingMode scaling_mode, std::vector<unsigned> max_size,
+              RocalResizeInterpolationType interpolation_type, std::vector<float>& mean,  std::vector<float>& std_dev, IntParam *mirror);
     vx_array get_dst_width() { return _dst_roi_width; }
     vx_array get_dst_height() { return _dst_roi_height;}
     vx_array get_src_width() { return _src_roi_width; }
     vx_array get_src_height() { return _src_roi_height; }
+    void adjust_out_roi_size();
     vx_array return_mirror(){ return _mirror.default_array();  }
 protected:
     void create_node() override;
     void update_node() override;
 private:
-    vx_array _dst_roi_width, _dst_roi_height;
-    std::vector<uint> _dest_width_val, _dest_height_val;
-    vx_array _mean_array, _std_dev_array;
-    std::vector<float> _mean;
-    std::vector<float> _std_dev; 
+    vx_array _mean_vx_array, _std_dev_vx_array, _mirror_vx_array, _dst_roi_width, _dst_roi_height;
+    std::vector<float> _mean, _std_dev;
+    int _interpolation_type;
     ParameterVX<int> _mirror;
     constexpr static int _mirror_range[2] = {0, 1};
+    RocalResizeScalingMode _scaling_mode;
+    unsigned _src_width, _src_height, _dst_width, _dst_height, _out_width, _out_height;
```

Comment:
Looks like there are too many variables used here for dims
`_src_width, _src_height, _dst_width, _dst_height, _out_width, _out_height;`
Are all of them actually needed?

Reply:
Yes